.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+
+[**.json]
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+
+[**.svg]
+trim_trailing_whitespace = false
```

Comment:
I believe we're missing an endline in here.
```suggestion
trim_trailing_whitespace = false

```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
The irony 😅
Did the exact same thing in another peoject, judt with prettier